compiler/x/amd64/codegen/OMRRealRegister.hpp

```diff
@@ -286,7 +286,7 @@ class OMR_EXTENSIBLE RealRegister : public OMR::X86::RealRegister
 
    void setSource2ndRegisterFieldInEVEX(uint8_t *opcodeByte)
       {
-      uint8_t regNum = ((_fullRegisterBinaryEncodings[_registerNumber].needsRexForByte << 3) | _fullRegisterBinaryEncodings[_registerNumber].id);
+      uint8_t regNum = ((_fullRegisterBinaryEncodings[_registerNumber].needsRexPlusRXB << 3) | _fullRegisterBinaryEncodings[_registerNumber].id);
```

Comment:
This change makes sense to me if you are updating the `vvvv` bits in P1.  However, the code below seems to suggest that P1 is one byte beyond the `opcodeByte`, so now I am confused.

Reply:
Perhaps `opcodeByte` is poorly named. `opcodeByte` points to EVEX P1 in the instruction where vvvv bits are, not the literal opcode value.

```uint8_t *evexP1 = opcodeByte + 1;``` should be P2